README.md

```diff
@@ -1,56 +1,60 @@
 # Node Feature Discovery Operator
- The Cluster Node Feature Discovery operator manages detection of hardware features and configuration in a Openshift cluster. The operator orchestrates all resources needed to run the NFD DaemonSet (Upstream: https://github.com/kubernetes-sigs/node-feature-discovery)
+ The Cluster Node Feature Discovery operator manages detection of hardware features and configuration in a Openshift cluster. The operator orchestrates all resources needed to run the NFD DaemonSet 
+ 
+## Upstream Project
 
-## Building the operator
-Checkout the sources
+[Node Feature Discovery](https://github.com/kubernetes-sigs/node-feature-discovery) – a Kubernetes add-on for detecting hardware features and system configuration.
 
-```bash
-$ git clone https://github.com/openshift/cluster-nfd-operator $GOPATH/src/github.com/openshift/cluster-nfd-operator
-```
-
-Build the operator image
+The [Node Feature Discovery](https://kubernetes-sigs.github.io/node-feature-discovery/stable/get-started/index.html) and [Node Feature Discovery operator](https://kubernetes-sigs.github.io/node-feature-discovery-operator/stable/get-started/index.html) are Upstream projects under the [kubernetes-Sigs](https://github.com/kubernetes-sigs) organization
 
-```bash
-IMAGE_REGISTRY=<my registry>
-make image
-```
+## Getting started with the Node Feature Discovery Operator
 
-Update the `Makefile` and edit `IMAGE`, `ORG` and `REGISTRY` one will need those later to update the operator manifest (`image: $(REGISTRY)/$(ORG)/$(cluster-nfd-operator:$(TAG))`).
+> Prerequisite: a running OpenShift cluster 4.6+ 
 
-Update the Dockerfile with the correct golang build image and the base image the operator will run with.
-
-## Manual deploy of the operator
-Checkout the sources
+Get the source code
 
+```bash
+git clone -b {{ site.release }} https://github.com/openshift/cluster-nfd-operator
```

Comment:
is this supposed to be transformed by a template engine? 
it seems it doesn't work for the README: https://github.com/ArangoGutierrez/cluster-nfd-operator/tree/devel/readme49

Reply:
good point, will fix